scripts/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "internal-scripts",
+  "scripts": {
+    "lint": "eslint --ext .js,.ts,.json, ."
```

Comment:
The root level `package.json` `lint` script also ran this task. This makes the `/scripts` directory its own lib from an Nx standpoint and moves the `lint` script here so that it can be cached the same way the other libs do